docs/en/connector-v2/sink/S3File.md

```diff
@@ -383,10 +404,93 @@ For orc file format simple config with `org.apache.hadoop.fs.s3a.SimpleAWSCreden
     access_key = "xxxxxxxxxxxxxxxxx"
     secret_key = "xxxxxxxxxxxxxxxxx"
     file_format_type = "orc"
+    schema_save_mode = "CREATE_SCHEMA_WHEN_NOT_EXIST"
+    data_save_mode="APPEND_DATA"
   }
 
 ```
 
+Multi-table writing and saveMode
+
+```
+env {
+"job.name"="SeaTunnel_job"
+"job.mode"=STREAMING
+}
+source {
+MySQL-CDC {
+    updateFieldType=null
+    autoCreateShow=true
+    autoCreate=false
+    autoTable=false
+    format=DEFAULT
+    "snapshot.split.size"=8096
+    "snapshot.fetch.size"=1024
+    "incremental.parallelism"=1
+    "connect.timeout.ms"=30000
+    "connect.max-retries"=3
+    "connection.pool.size"=6
+    "chunk-key.even-distribution.factor.lower-bound"=0.05
+    "chunk-key.even-distribution.factor.upper-bound"=100
+    "sample-sharding.threshold"=1000
+    "inverse-sampling.rate"=1000
+    "startup.mode"=INITIAL
+    "exactly_once"="true"
+    "stop.mode"=NEVER
+    parallelism=1
+    "result_table_name"=Table11519548644512
+    "dag-parsing.mode"=MULTIPLEX
+    catalog {
+        factory=Mysql
+    }
+    database-names=[
+        "wls_t1"
+    ]
+    table-names=[
+        "wls_t1.mysqlcdc_to_s3_t3",
+        "wls_t1.mysqlcdc_to_s3_t4",
+        "wls_t1.mysqlcdc_to_s3_t5",
+        "wls_t1.mysqlcdc_to_s3_t1",
+        "wls_t1.mysqlcdc_to_s3_t2"
+    ]
+    password="xxxxxx"
+    username="xxxxxxxxxxxxx"
+    base-url="jdbc:mysql://localhost:3306/qa_source"
+    server-time-zone=UTC
+}
+}
+transform {
+}
+sink {
+S3File {
+    updateFieldType=null
```

Comment:
It seems `updateFieldType`,`autoCreateShow `,`autoTable ` is not the options of S3 Sink Connector.
And `partition_dir_expression ` `is_partition_field_write_in_file ` are no need to config when `have_partition = false`.

Please check the examples config, It needs to be sufficiently correct.